### ISSUE TITLE
reqesting the ability to ignore spycrabs

### ISSUE THREAD

**Issue by joeyjumper94** [nullworks/cathook#949]
much like how hoovies are ignored because of them being friendly,
i suggest the ability to ignore spycrabs cause they are also associated with friendlies.

**Comment by LightPower392** [comment]
Would this just be the hoovy check but with a disguise kit instead of sandvich?

**Comment by loganx1** [comment]
Yes, in order to at least get basic support you literally have to change one line of code, which is pretty easy to do.
